src/tests/ftest/util/soak_test_base.py

```diff
@@ -559,7 +701,7 @@ def run_soak(self, test_param):
         resv_bytes = self.params.get("resv_bytes", test_param + "*", 500000000)
         ignore_soak_errors = self.params.get("ignore_soak_errors", test_param + "*", False)
         self.enable_il = self.params.get("enable_intercept_lib", test_param + "*", False)
-        self.sudo_cmd = "sudo" if enable_sudo else ""
+        self.sudo_cmd = "sudo -n" if enable_sudo else ""
```

Comment:
We should consider replacing uses of `self.sudo_cmd` with `command_as_user()` in the future to simplify maintenance like this.